src/patternfly/demos/Tabs/examples/Tabs.md

```diff
@@ -0,0 +1,814 @@
+---
+id: 'Tabs'
+section: demos
```

Comment:
so it shows up as the demos tab on the component, not in the demos section in the vertical nav

```suggestion
section: components
```